### ISSUE TITLE
[ANNOUNCEMENT] - RevoGain - an alternative that actually works!

### ISSUE THREAD

**Issue by vladmihalcea** [doino-gretchenliev/revolut-stocks#31]
Revolut statements are very hard to parse, and that's not all. There are actually many more issues, like:

- stocks undergo splits and reverse splits so you need to take that into consideration when calculating the base cost
- you can buy crypto with USD, GBP, and EUR, but gains have to be converted to a base currency when doing the report
- the crypto Excel/CSV statement does not even contain the buying/selling price so you need the historical price for that
- you need the historical price for Gold and Silver if you trade those as well
- you can buy BTC with EUR, then sell it to ETH, and then buy USD. All those operations are taxable.
- you need to calculate the dividend cost
- as a Revolut customer, you might want to generate your report in your local currency (e.g., 'EUR', 'GBP', 'CHF', 'USD', 'DKK', 'SEK', 'NOK', 'PLN', 'CZK', 'HRK', 'HUF', 'BGN', 'RON)

Trying to make a sense out of all that could take you days or just a minute if you use [RevoGain](https://revogain.com/).

**Comment by doino-gretchenliev** [comment]
1. All types of splits are supported.
2. Crypto calculations are not supported.
3. The new Revolut CSV statements will be supported before the next tax season.
4. Commodities are not supported.
5. Dividend costs are fully supported.
6. Currently the calculator works with BGN, according to Bulgarian tax lows.

1. RevoGain doesn't seem to work with any EU tax jurisdiction.
2. My calculator is **free** and always remains **free**.
